### PR TITLE
Wagtail 4.1 - 5.1

### PR DESCRIPTION
**Pull request by katdom13**
### Release notes:
[Wagtail 4.1](https://docs.wagtail.org/en/stable/releases/4.1.html)
[Wagtail 4.2](https://docs.wagtail.org/en/stable/releases/4.2.html)
[Wagtail 5.0](https://docs.wagtail.org/en/stable/releases/5.0.html)
[Wagtail 5.1](https://docs.wagtail.org/en/stable/releases/5.1.html)

### CODE REVIEW

**Comment by seb-b** [comment]
I appreciate the tests but this is a very simple project, it just adds a template tag for an included js file (which is fairly out of date at the moment). Mainly out of curiosity but what was the motivation behind all this effort? This project is probably going to be deprecated because of the [included accessibility checker](https://docs.wagtail.org/en/stable/releases/4.2.html#accessibility-checker-integration) since 4.2

File: setup.py [hunk 1]
```diff
@@ -13,6 +13,18 @@
     use_setuptools()
     from setuptools import setup, find_packages
 
+install_requires = [
+    'wagtail>=4.1',
```
Comment (seb-b) [inline setup.py:17]: is there some reason this was added? This should in theory work in any wagtail version (or even non wagtail projects really)

Comment (katdom13) [reply setup.py:17]: It's just that Wagtail versions before 4.1 have reached their end of life: https://endoflife.date/wagtail

**Comment by katdom13** [comment]
> I appreciate the tests but this is a very simple project, it just adds a template tag for an included js file (which is fairly out of date at the moment). Mainly out of curiosity but what was the motivation behind all this effort? This project is probably going to be deprecated because of the [included accessibility checker](https://docs.wagtail.org/en/stable/releases/4.2.html#accessibility-checker-integration) since 4.2

Hi @seb-b ,
We have an internal effort to update packages that are actively being used on some of our websites and this is one of them.
Though I'll make sure to raise this and update the sites accordingly to use the built-in accessibility checker instead.
Thanks for looking at this.

Cheers!

**Comment by thibaudcolas** [comment]
@seb-b what do you think of moving this package to [@wagtail-nest](https://github.com/wagtail-nest) rather than deprecating it, so you can share the maintenance effort with others?

From my perspective though I’d like to see what `wagtail-accessibility` offers inside Wagtail core eventually, right now what we have in core is way too basic for people who want the functionality provided in this package.

**Comment by seb-b** [comment]
Hi @thibaudcolas, I reached out on slack but didn't hear back. I'd be happy to move this project over to nest, we no longer use it and don't have plans on maintaining it

**Comment by thibaudcolas** [comment]
Sorry @seb-b, I was away for a few weeks and let my inbox pile up. I’ve just messaged you, and sent you an invitation to https://github.com/wagtail-nest so you can initiate the transfer if that still works for you.

**Comment by thibaudcolas** [comment]
Merged in dc4a4b81d2458f00912c301494d3ba64dbf597c3, thank you!

**Pull request closed without merging**